plugins/out_s3/s3.c

```diff
@@ -252,6 +253,128 @@ static flb_sds_t concat_path(char *p1, char *p2)
     return dir;
 }
 
+/* Reads in index value from metadata file and sets seq_index to value */
+static int read_seq_index(char *seq_index_file, uint64_t *seq_index)
+{
+    FILE *fp;
+    int ret;
+
+    fp = fopen(seq_index_file, "r");
+    if (fp == NULL) {
+        return -1;
+    }
+
+    ret = fscanf(fp, "%"PRIu64, seq_index);
```

Comment:
typo or is this is C thing that I don't know?

Reply:
Oh I see, it is a C thing that I didn't know about: https://stackoverflow.com/questions/16859500/mmh-who-are-you-priu64